bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/ProfileAdvisor.java

```diff
@@ -41,4 +42,20 @@
     @Nullable
     ProfileTypeUID getSuggestedProfileTypeUID(Channel channel, @Nullable String itemType);
 
+    /**
+     * Suggest a custom profile for a given {@link ChannelType} (and potentially also the itemType).
+     *
+     * Please note:
+     * <ul>
+     * <li>This will override any default behavior
+     * <li>A "profile" configuration on the link will override this suggestion
+     * </ul>
+     *
+     * @param channelType the {@link ChannelType} of the linked channel
+     * @param itemType the linked itemType (not applicable for trigger channels)
+     * @return the profile identifier or {@code null} if this advisor
```

Comment:
"if this advisor" ...? something is missing here

Reply:
This is just a copy of the original method javaDoc. We have to ask @sjka aboiut the missing text here.

Reply:
"... does not have any advice"